### DISCUSSION TITLE
Edit or delete any message

### DISCUSSION THREAD

**Discussion by 0xCA** [Taewan-P/gpt_mobile#112] [Ideas]
Please implement two additional buttons for each message, similar to openai playground interface, that allows to modify chat history in any way

**Comment by Taewan-P** [comment]
This is solved in #119. Thanks

**Reply by 0xCA** [reply]
> This is solved in #119. Thanks

From what I can see, it only allows edits, but not deletions of messages

**Reply by Taewan-P** [reply]
Deletion of messages will not be implemented because it is not necessary. You can just edit the previous message and its prior messages will be deleted.
